ls1intum/Artemis
# `General`: Improve exercise header

### Checklist
#### General


#### Client

### Motivation and Context


The exercise header provides important information about the exercise but should not take up too much space while doing so. This PR improves this further.

### Description

The exercise header now provides information about the achieved points in the exercise.
The left column of the header was often longer than the right column. To counter this two short rows of the left column are now in the same line when there is enough width.
The tooltips explaining the assessment types were improved.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Students
- 1 Exercise (Modeling or Text)
- 1 Programming Exercise 

1. Log in to Artemis
2. Create the exercise
3. Participate in the exercises as student
4. Grade the modeling/text exercise
5. Check the page showing the assessment as student and check the header there as well
6. Check the header of both exercises starting in step 3

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots


![Bildschirmfoto 2022-09-28 um 20 42 01](https://user-images.githubusercontent.com/38322605/192863074-088d8341-742c-4980-997e-dfffb6098192.png)
